### PR TITLE
Color log lines in fauna local command.

### PR DESCRIPTION
**Pull request by cleve-fauna**
## Problem

We don't color log lines. This can make them harder to read and look less appealing.
## Solution

Add a highlighter for log lines. Apply it to the local command.

## Result

The local command colors logs. Other commands can add such coloring as needed.

## Testing
I ran the command and saw color:
Screenshot 2024-12-13 at 11 13 27 AM

### CODE REVIEW

File: src/lib/docker-containers.mjs [hunk 1]
```diff
@@ -392,10 +388,19 @@ async function waitForHealthCheck({
...
   throw new CommandError(
     `[HealthCheck] Fauna at ${url} is not ready after ${maxAttempts} attempts. Consider increasing --interval or --maxAttempts.`,
   );
 }
+
+/**
+ * Outputs to stderr.
+ * @param {string} log The log
+ */
+function stderr(log) {
```
Comment (ptpaterson) [inline src/lib/docker-containers.mjs:403]: Can we avoid the mutable global `color` by accepting it as an argument in `stderr`?

Comment (cleve-fauna) [reply src/lib/docker-containers.mjs:403]: then i would have to pass color to every private method in this module.

I'd prefer to memoize it.

Comment (ptpaterson) [reply src/lib/docker-containers.mjs:403]: Understood. I realize I prefer verbosity over pit traps like global variables. You could curry the color `const stderrColored = stderr.bind(null, color)` if you want, but you probably don't 😜

**Review by ptpaterson — APPROVED** [review]

File: src/lib/docker-containers.mjs (hunk 1 above)
Comment (ptpaterson) [reply src/lib/docker-containers.mjs:403]: no that's dumb in this case ignore that.

But you could declare stderr function within `ensureContainerRunning` to close over the color provided.

```js
export async function ensureContainerRunning({
  containerName,
  hostIp,
  hostPort,
  containerPort,
  pull,
  interval,
  maxAttempts,
  color,
}) {
  function stderr(log) {  
    const logger = container.resolve("logger");
    logger.stderr(colorize(log, { format: Format.LOG, color }));
  }

  // ...
```

Comment (cleve-fauna) [reply src/lib/docker-containers.mjs:403]: then i'd have to pass that around.

Comment (ptpaterson) [reply src/lib/docker-containers.mjs:403]: misread the diff, it's needed in several functions 👍🏻

**Review by mwilde345 — APPROVED** [review]

**Pull request merged**